app/views/games/show.html.erb

```diff
@@ -28,95 +63,96 @@
-        <div class="white left"><%= render_piece(6,2) %></div>
-        <div class="black left"><%= render_piece(7,2) %></div>
-      </div>
-      <div class="row">
-        <div class="black left"><%= render_piece(0,1) %></div>
-        <div class="white left"><%= render_piece(1,1) %></div>
-        <div class="black left"><%= render_piece(2,1) %></div>
-        <div class="white left"><%= render_piece(3,1) %></div>
-        <div class="black left"><%= render_piece(4,1) %></div>
-        <div class="white left"><%= render_piece(5,1) %></div>
-        <div class="black left"><%= render_piece(6,1) %></div>
-        <div class="white left"><%= render_piece(7,1) %></div>
-      </div>
+        <div class="row">
+          <div class="white left"><%= render_piece(0,2) %></div>
+          <div class="black left"><%= render_piece(1,2) %></div>
+          <div class="white left"><%= render_piece(2,2) %></div>
+          <div class="black left"><%= render_piece(3,2) %></div>
+          <div class="white left"><%= render_piece(4,2) %></div>
+          <div class="black left"><%= render_piece(5,2) %></div>
+          <div class="white left"><%= render_piece(6,2) %></div>
+          <div class="black left"><%= render_piece(7,2) %></div>
+        </div>
+        <div class="row">
+          <div class="black left"><%= render_piece(0,1) %></div>
+          <div class="white left"><%= render_piece(1,1) %></div>
+          <div class="black left"><%= render_piece(2,1) %></div>
+          <div class="white left"><%= render_piece(3,1) %></div>
+          <div class="black left"><%= render_piece(4,1) %></div>
+          <div class="white left"><%= render_piece(5,1) %></div>
+          <div class="black left"><%= render_piece(6,1) %></div>
+          <div class="white left"><%= render_piece(7,1) %></div>
+        </div>
 
-      <div class="row bottom">
-        <div class="white left"><%= render_piece(0,0) %></div>
-        <div class="black left"><%= render_piece(1,0) %></div>
-        <div class="white left"><%= render_piece(2,0) %></div>
-        <div class="black left"><%= render_piece(3,0) %></div>
-        <div class="white left"><%= render_piece(4,0) %></div>
-        <div class="black left"><%= render_piece(5,0) %></div>
-        <div class="white left"><%= render_piece(6,0) %></div>
-        <div class="black left"><%= render_piece(7,0) %></div>
+        <div class="row bottom">
+          <div class="white left"><%= render_piece(0,0) %></div>
+          <div class="black left"><%= render_piece(1,0) %></div>
+          <div class="white left"><%= render_piece(2,0) %></div>
+          <div class="black left"><%= render_piece(3,0) %></div>
+          <div class="white left"><%= render_piece(4,0) %></div>
+          <div class="black left"><%= render_piece(5,0) %></div>
+          <div class="white left"><%= render_piece(6,0) %></div>
+          <div class="black left"><%= render_piece(7,0) %></div>
```

Comment:
should i be doing this in the game view or the helper method for render piece and have it loop through and assign a data url to each?